### PR TITLE
Wk2 407 isa subscription documentation

### PR DESCRIPTION
**Pull request by tom-meader-wk**

### CODE REVIEW

File: docs/isa-subscriptions/Basics.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Any account which is a type of ISA will have a subscription. An ISA Subscription keeps track of how much a user has subscribed towards their ISA account in a given tax year. This includes when a user deposits money into their portfolio, as well as a user transferring an ISA from a previous provider. Note that depending on various factors, it may take some time for the subscription amount to be updated in the case of an ISA transfer.
+
+It's worth noting that not all changes to a portfolio mean a change to the corresponding ISA Subscription. The following do not change the value of an ISA Subscription:
+
+- Dividends
+- A change in value of the assets
+- Buys and sells
+- Charges
+- Withdrawals
```
Comment (niall-maloney) [inline docs/isa-subscriptions/Basics.md:16]: This is only true for non-flexible ISAs

File: docs/isa-subscriptions/Basics.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+tags: [ISA Subscriptions]
+stoplight-id: 8738880941b84
+---
+
+# ISA Subscription Basics
+
+Any account which is a type of ISA will have a subscription. An ISA Subscription keeps track of how much a user has subscribed towards their ISA account in a given tax year. This includes when a user deposits money into their portfolio, as well as a user transferring an ISA from a previous provider. Note that depending on various factors, it may take some time for the subscription amount to be updated in the case of an ISA transfer.
```
Comment (niall-maloney) [inline docs/isa-subscriptions/Basics.md:8]: What about JISAs?

**Comment by adonozo-wk** [comment]
In a stoplight demo, `/custom-functions` and `spectral.json` were added to the gitignore. Maybe we should do the same?
https://github.com/stoplightio/elements/blob/6f26e08fa8ae85cd95ced7e73e3391be637a94be/.gitignore#L49

File: docs/isa-subscriptions/Basics.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+tags: [ISA Subscriptions]
+stoplight-id: 8738880941b84
+---
+
+# ISA Subscription Basics
+
+Any account which is a type of ISA or JISA will have a subscription. A subscription keeps track of how much a user has subscribed towards their account in a given tax year. This includes money is deposited into their portfolio(s), as well as a user transferring an ISA/JISA from a previous provider. Note that depending on various factors, it may take some time for the subscription amount to be updated in the case of a transfer.
+
+It's worth noting that not all changes to a portfolio mean a change to the corresponding subscription. The following do not change the value of a subscription:
```
Comment (niall-maloney) [inline docs/isa-subscriptions/Basics.md:10]:
```suggestion
Be aware that not all changes to a portfolio mean a change to the corresponding subscription. The following do not change the value of a subscription:
```

**Pull request merged**